code/controllers/subsystem/mapping.dm

```diff
@@ -768,6 +768,8 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 		GLOB.fullbright_overlays += create_fullbright_overlay(offset)
 		GLOB.cryo_overlays_cover_on += create_cryo_overlay(offset, "cover-on")
 		GLOB.cryo_overlays_cover_off += create_cryo_overlay(offset, "cover-off")
+		GLOB.cleaning_bubbles_lower += generate_bubbles(GAME_PLANE, offset)
+		GLOB.cleaning_bubbles_higher += generate_bubbles(ABOVE_GAME_PLANE, offset)
```

Comment:
I know this was from the other PR but what is this whole proc? This looks like the kind of thing that is going to trip feature coders up, that they'll have to edit the mapping subsystem to get...something?

Reply:
I wish there was a better way of handling it.
Basically, people sometimes want to store global versions of mutable appearances (though in all honesty this is hardly needed most of the time)
We need to generate a new mutable appearance each time, because they tend to involve planes. So the list gets indexed by the offset to use, and we generate a new MA each time somethin increases the offset.

Lemme do a pass for dumb uses of this

Reply:
The only reason to do it at this scale is like, KINDA BARELY cpu time things, but it's like, nothing.
If we really cared we could setup a proc to autocache these, and assert something about their mutability afterwards.